biojava-core/src/main/java/org/biojava/nbio/core/search/io/blast/BlastHitBuilder.java

```diff
@@ -33,49 +34,49 @@
  * 
  * @author Paolo Pavan
  */
-public class BlastHitBuilder {
+public class BlastHitBuilder<S extends Sequence<C>, C extends Compound> {
     private int hitNum;
     private String hitId;
     private String hitDef;
     private String hitAccession;
     private int hitLen;
-    private Sequence hitSequence;
-    private List<Hsp> hsps;
+    private Sequence<?> hitSequence;
```

Comment:
If you're going to make BlastHitBuilder parametric then you better use those parameters here. Should be an `S`.